consensus/sha3pow/Cargo.toml

```diff
@@ -18,8 +18,8 @@ compatibility_version = "2.0.0-dev"
 parity-scale-codec = '1.3.0'
 sha3 = "0.8"
 rand = { version = "0.7", features = ["small_rng"] }
-sc-consensus-pow = { version = '0.8.0-dev', git = 'https://github.com/paritytech/substrate', rev = 'cd305af2a9f8ec7d13081b6dc6807014b207dab8' }
```

Comment:
all of these rev need to be updated to reflect the new version (?)

Reply:
The commit you're commenting on here, was the prep work to get ready for RC3 release. I first updated the dependencies to the github commit that corresponds to the RC3 tag, and made the code compile.

Once that was done, I then switched the dependencies from github to crates.io